### PR TITLE
Trying to get maven artifact repo working

### PR DESCRIPTION
**Pull request by jadecarino**
Signed-off-by: Jade Carino <jade.carino@ibm.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]
Looks good to me. Perhaps best to remove the commented-out bits ?
Unless you are leaving them in for a reason ?

File: pipelines/roles/maven-repo-role-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#
+# Copyright contributors to the Galasa project
```
Comment (techcobweb) [inline pipelines/roles/maven-repo-role-binding.yaml:2]: Has this Copyright message been approved by legal ?
Normally I thought they had dates in..etc.

**Comment by jadecarino** [comment]
Hi Mike, thanks, unfortunately we are still not quite there with what we are trying to do - I had left the comments in on purpose as Aaron will be continuing with this tomorrow while I am at university for the day and I thought the comments might be useful. I have done some further work on this this evening and have pushed those changes to this PR, if you wanted to take another look tomorrow to help Aaron.

File: pipelines/roles/maven-repo-role-binding.yaml (hunk 1 above)
Comment (jadecarino) [reply pipelines/roles/maven-repo-role-binding.yaml]: @hobbit1983 Can you provide some insight on this please? I assumed the copyright block had been approved as we have been using it for some time now

**Comment by jadecarino** [comment]
Have to close this PR and open a fresh one as the PR builds do not trigger from the 'sychronize' action of a PR

File: pipelines/roles/maven-repo-role-binding.yaml (hunk 1 above)
Comment (hobbit1983) [reply pipelines/roles/maven-repo-role-binding.yaml]: @jadecarino @techcobweb yes this has changed quite a few times. The last review of the copyright headers by legal resulted in the one we are now using

**Pull request closed without merging**